airflow/utils/json.py

```diff
@@ -44,7 +44,12 @@ def loads(self, s: str | bytes, **kwargs):
 
 
 class WebEncoder(json.JSONEncoder):
-    """This encodes values into a web understandable format. There is no deserializer"""
+    """
+    This encodes values into a web understandable format. There is no deserializer.
+    It parses datetime, dates, Decimal and bytes. In order to parse the custom
+    classes and the other types, and since it's just to show the result in the UI,
+    we return repr(object) for everything else.
+    """
```

Comment:
```suggestion
    """This encodes values into a web understandable format. There is no deserializer.

    This parses datetime, dates, Decimal and bytes. In order to parse the custom
    classes and the other types, and since it's just to show the result in the UI,
    we return repr(object) for everything else.
    """
```